domains/andruudev.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "andruudev",
+    "email": "andruudev@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://andruudev.github.io/portfolio/"
```

`CNAME` record must contain domain name only.
```suggestion
    "CNAME": "andruudev.github.io"
```